.circleci/deploy-testnet.sh

```diff
@@ -1,16 +1,27 @@
 #!/bin/bash -e
 
+# Installing aws cli
+echo "Installing AWS CLI"
+sudo apt-get update
+sudo apt-get install -y python-dev
+sudo apt-get install -y python-pip
+sudo pip install awscli
+
+aws --version
+
 touch $BASH_ENV
 curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
-
 export NVM_DIR="/opt/circleci/.nvm" && . $NVM_DIR/nvm.sh && nvm install v10.14.1 && nvm use v10.14.1
+export COMMIT_HASH=$(./docker/hash.sh)
 
-echo $TESTNET_SSH_PUBLIC_KEY > ~/.ssh/id_rsa.pub
+curl -O https://s3.eu-central-1.amazonaws.com/boyar-ci/boyar/config.json
+node .circleci/testnet-deploy-tag.js $COMMIT_HASH
 
-export COMMIT_HASH=$(./docker/hash.sh)
+aws s3 cp --acl public-read config.json s3://boyar-ci/boyar/config.json
```

Comment:
Please use nebula instead. We have all the API that is needed for that.

Reply:
The code in Nebula can generate a complete configuration and requires the complete set of network  JSON files. 
If I do that here - I need to copy or add a git submodule of testnet configuration in here (which is a private repo)

I was aiming to try and only change the version deployed to the chains in the file instead of changing the entire file. I can do that - let me know how do you want to solve sharing the JSONs required to run these APIs in nebula.

The `aws s3 cp` calls are still required as far as I can remember.

Reply:
Ok, let's keep it that way for now.